pkg/virt-handler/vm.go

```diff
@@ -441,6 +442,28 @@ func (d *VirtualMachineController) execute(key string) error {
 	return nil
 }
 
+func (d *VirtualMachineController) replacePVCByHostDisk(vmi *v1.VirtualMachineInstance) error {
+	// if PVC is defined then it is replaced by HostDisk
+	// PersistenVolumeClaim is mounted into pod as directory from node filesystem
+	// it is not attached as independent block device
+	for i := range vmi.Spec.Volumes {
```

Comment:
Hm, I did this on purpose, few lines below I am updating volume source structure (PVC is changing to HostDisk). My point is that I would have a different memory address and would have to do more to update the volume structure, because of ```range``` is copying values:
```
&vmi.Spec.Volumes[i] != &volume 
```

Although I am not so experienced with golang, so after this explanation do you still think I should change this? Thx.

Reply:
I see do you mean this one `volumeSource.PersistentVolumeClaim = nil`, works for me

Reply:
Yup, and this one: 
```
volumeSource.HostDisk = &v1.HostDisk{
				Path:     hostdisk.GetPVCDiskImgPath(vmi.Spec.Volumes[i].Name),
				Type:     v1.HostDiskExistsOrCreate,
				Capacity: pvcSize,
			}
```